src/fam-api/fam.cpp

```diff
@@ -943,11 +943,6 @@ configFileParams fam::Impl_::get_info_from_config_file(std::string filename) {
         try {
 
             std::string allocator = info->get_key_value("openfam_model");
-            options["openfam_model"] =
```

Comment:
Are we not reading using "openfam_model" anywhere during initialization of OpenFAM?